src/components/scss/nav.scss

```diff
@@ -0,0 +1,40 @@
+nav {
+  background-color: #f60;
+  display: flex;
+  width: 100%;
+  height: 150px;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  border-radius: 0 0 15px 15px;
+  border-bottom: 1px solid white;
+  position: absolute;
+  top: 0;
+  z-index: 4;
+
+  .location-path {
+    display: flex;
+
+    .back {
+      background-color: transparent;
+      color: white;
+      border: none;
+      font-weight: bold;
+      font-size: 50px;
+      font-family: 'lato', sans-serif;
+      text-align: center;
+    }
+  }
+}
+
+@media only screen and (max-width: 619px) {
+  .back {
+    font-size: 30px !important;
+  }
+}
```

Comment:
- [ ] **[OPTIONAL]** Let's not overuse the `!important` rule as it is not a good practice to have many `!important` rules.